packages/flutter_test/lib/src/controller.dart

```diff
@@ -477,6 +477,14 @@ abstract class WidgetController {
   /// system identifies the gesture as a fling, consider using [fling] instead.
   ///
   /// {@template flutter.flutter_test.drag}
+  /// If `duration` is not `null`, this is a convenience wrap of
+  /// [handlePointerEventRecord], with the input lasts for `duration` at the
+  /// given `frequency` (Hz).
```

Comment:
Nit: consider reusing the documentations from `FlutterDriver` such as
```
[duration] specifies the length of the action. 

The move events are generated at a given [frequency] in Hz (or events per second). It defaults to 60Hz.

If [duration] is null, input events are assumed to happen at once, and [frequency] is ignored.

See also [LiveTestWidgetsFlutterBindingFramePolicy.benchmarkLive] for more accurate time control.
```

Reply:
Done